### ISSUE TITLE
Can tdnf install pre-downloaded local RPM Packages?

### ISSUE THREAD

**Issue by KohliDev** [vmware/tdnf#77]
Please guide on how tdnf can install local pre-downloaded rpm packages.

https://communities.vmware.com/thread/519734

**Comment by keerthanakalyan** [comment]
You can create a local repository and try using tdnf. Please refer to this https://github.com/vmware/tdnf/wiki#adding-a-new-repository

**Comment by pixdrift** [comment]
It appears that tdnf/yum implementation doesn't support `localinstall` or a local file target for install. Is there a plan to implement this in future?

```
yum localinstall package.rpm
No such command: localinstall. Please use /usr/bin/tdnf --help
```

**Comment by oliverkurth** [comment]
Implemented with 86cf432f04158a7e4ca2552906d720d9d28403d6 . It's not `localinstall`, just `tdnf install /path/to/file-1.2.3.rpm`.
